extras/connection-pool/src/test/java/org/glassfish/grizzly/connectionpool/SingleEndPointPoolTest.java

```diff
@@ -622,5 +624,17 @@ public void testKeepAliveZero() throws Exception {
             pool.close();
             transport.shutdownNow();
         }
-    }    
+    }
+
+    private static int findFreePort() {
+        try {
+            ServerSocket dummySocket = new ServerSocket(0);
+            int freePort = dummySocket.getLocalPort();
+            dummySocket.setReuseAddress(true);
+            dummySocket.close();
+            return freePort;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
Is it possible to reuse the one from the utils in the http module? or it doesn't have visibility?